WordPress/src/main/java/org/wordpress/android/ui/comments/CommentsDetailActivity.java

```diff
@@ -94,6 +83,22 @@ public void onCreate(Bundle savedInstanceState) {
 
         setContentView(R.layout.comments_detail_activity);
 
+        OnBackPressedCallback callback = new OnBackPressedCallback(true) {
+            @Override
+            public void handleOnBackPressed() {
+                CollapseFullScreenDialogFragment fragment = (CollapseFullScreenDialogFragment)
+                        getSupportFragmentManager().findFragmentByTag(CollapseFullScreenDialogFragment.TAG);
+
+                if (fragment != null) {
+                    fragment.onBackPressed();
```

Comment:
Minor (🔍): Consider renaming this publicly available, but not `@Override`, method to something else. It can become misleading and others might incorrectly assume that this is indeed an `@Override onBackPressed` method that needs to be used as such from all these 3 activities that are calling it, I know because I did just that at first... 😅

Maybe a better name for that method would be something like `checkAddedAndCollapse()`? 🤔 

PS: Same applies to `FullScreenDialogFragment`, but this method there seems to be unused as well... 🤷

Reply:
I was also confused and checked if `onBackPressed` is an override function. 😄 It will be a good action to rename it. 
What about renaming it to `collapse()`? The `checkAdded...` part doesn't seem to have much meaning to me.

I believe we can remove `onBackPressed` from `FullScreenDialogFragment` as it is currently unused. I see no reason not to remove it.

Reply:
👋 @irfano !

> What about renaming it to collapse()? The checkAdded... part doesn't seem to have much meaning to me.

Yea, let's better rename it to something as simple as `collapse()`. 👍 💯 

> I believe we can remove onBackPressed from FullScreenDialogFragment as it is currently unused. I see no reason not to remove it.

Coolio, let's remove it then, I agree! 👍 💯

Reply:
Done: fb7a69c9b73653838f56ff7d1bd3ffc3de2bbb56

Reply:
Awesome! 🚀